### ISSUE TITLE
Non-comprehensive instruction to get started!

### ISSUE THREAD

**Issue by absholute** [JohnCoates/CSSketch#54]
Hey John,
Can you please be specific with your instruction to use it? We can't get it to work. We installed in various ways (Sketch toolbox, Sketch Kit, Directly from GIT using your latest release), none worked.

We're lacking instruction on how to use them first time. Here goes basic questions that should be covered:
- Opened your example files
- Click the artboard
- Go back to menu bar > Sketch > Plugins > CSSSketch > Set Page Stylesheet (Selecting either CSS, SASS or LESS) the dialog keeps popping back up, does nothing at all. Seems that we cant link
- The other option does nothing too for Layout layer
- Whats the process on getting it hooked up to CSS? It's quite confusing. Please put instruction in layman term. Remember your audience are mainly designers, not developers and please don't assume we all know. The best way to tackle this is to do youtube tutorial. Thanks!

**Comment by absholute** [comment]
I'm pretty sure this would be an easy answer by anyone who has used it or developed it?

**Comment by julienpecorino** [comment]
Hi,
If you are interested I have written a little article on how to use
CSSketch for a wide UI project:
https://blog.trailapp.com/sketch-css-in-search-of-the-perfec
t-ui-workflow-8be56b24c984#.ff2otwzbq

I have heavily worked with this plugin in my last start-up to design their
software and maintain their style guide.

The plugin works best with an old version of sketch ---> 3.6.
Below in the comments you can find the link to the demo files and which
version to install,
here the direct link to the demo:
julien-pecorino.com/projects/cssketch-demo%202.zip

Hope this can help!

Thanks to the creator of CSSketch,
I wish to have more dev skills to help improving the plugin...

Best

*Julien Pecorino*
Senior Interactive Designer
—
portfolio:   julien-pecorino.com
mail:        julien.pecorino@gmail.com
skype:     julien.pecorino
mobile:    0033 675 790 038
—


On 10 Jan 2017 9:59 am, "absholute" <notifications@github.com> wrote:

I'm pretty sure this would be an easy answer by anyone who has used it or
developed it?

—
You are receiving this because you are subscribed to this thread.
Reply to this email directly, view it on GitHub
<https://github.com/JohnCoates/CSSketch/issues/54#issuecomment-271520835>,
or mute the thread

**Comment by absholute** [comment]
Thanks for taking the time to this Julien! Appreciate your reply. 

"The plugin works best with an old version of sketch ---> 3.6." <<-- This might be the missing piece for me. I'm on 41.2 (current latest). I'll try and install the older version and see what would impact my current project. Will keep you posted! Cheers!

**Comment by absholute** [comment]
Unfortunately, I won't be able to use outdated sketch due to other plugins that are required. This is a massive drawback to such awesome plugin. I'll be using Flex layout instead which works well. Cheers!
